lms/djangoapps/teams/views.py

```diff
@@ -0,0 +1,508 @@
+
+                * country: Optionally specifies which country the team is
+                  associated with.
+
+                * language: Optionally specifies which language the team is
+                  associated with.
+
+                * membership: A list of the users that are members of the team.
+                  See membership endpoint for more detail.
+
+            For all text fields, clients rendering the values should take care
+            to HTML escape them to avoid script injections, as the data is
+            stored exactly as specified. The intention is that plain text is
+            supported, not HTML.
+
+            If the user is not logged in and enrolled in the course specified by
+            course_id or is not staff, a 403 error is returned.
+
+            If the specified course_id is not valid or the user attempts to
+            use an unsupported query parameter, a 400 error is returned.
+
+            If the response does not exist, a 404 error is returned. For
+            example, the course_id may not reference a real course or the page
+            number may be beyond the last page.
+
+        **Response Values for POST**
+
+            Any logged in user who has verified their email address can create
+            a team. The format mirrors that of a GET for an individual team,
+            but does not include the id, is_active, date_created, or membership
+            fields. id is automatically computed based on name.
+
+            If the user is not logged in or is not enrolled in the course, a
+            403 error is returned.
+
+            If the course_id is not valid or extra fields are included in the
+            request, a 400 error is returned.
+
+            If the specified course does not exist, a 404 error is returned.
+    """
+
+    # SessionAuthentication must come first to return a 403 for unauthenticated users
+    authentication_classes = (SessionAuthentication, OAuth2Authentication)
+    permission_classes = (permissions.IsAuthenticated,)
+
+    paginate_by = 10
+    paginate_by_param = 'page_size'
+    pagination_serializer_class = PaginationSerializer
+    serializer_class = CourseTeamSerializer
+
+    def get_serializer_context(self):
+        """Adds expand information from query parameters to the serializer context to support expandable fields."""
+        result = super(TeamsListView, self).get_serializer_context()
+        result['expand'] = [x for x in self.request.QUERY_PARAMS.get('expand', '').split(',') if x]
+        return result
+
+    def get(self, request):
+        """GET /api/team/v0/teams/"""
+        result_filter = {
+            'is_active': True
+        }
+
+        if 'course_id' in request.QUERY_PARAMS:
+            course_id_string = request.QUERY_PARAMS['course_id']
+            try:
+                course_key = CourseKey.from_string(course_id_string)
+                get_course(course_key)
```

Comment:
Nit: I assume you are doing this just to make sure the course does exist. Perhaps a comment? it seems strange because you are calling get_course, but not using any return value.